frontend/src/scenes/data-warehouse/editor/multitabEditorLogic.tsx

```diff
@@ -332,7 +343,6 @@ export const multitabEditorLogic = kea<multitabEditorLogicType>([
         ],
         allTabs: [
             [] as QueryTab[],
-            { persist: true },
```

Comment:
another gotcha here is if you remove the persistence on allTabs in the kea logic, there's no migration path from allTabs to indexed db. The key being used in indexdb is different than what the key the kea logic uses under the hood here. So upon merge, allTabs will be empty.

The consequence of ^ is if you have a query that has query variables, they'll be purged and you'll see an error. Clearest way to see this is on `main` write the query with a query variable like `SELECT * FROM events where name = {variables.test}`, make sure the variable exists. Then switch to this branch and you'll see an error